src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const HomePage: React.FC = () => {
+  return (
+    <div className="container">
+      <h1 className="title">Home page</h1>
+    </div>
+  );
+};
```

Comment:
you dont need to use return here, check also another components

Reply:
Ohh, true 🥲, thanks 🙂